packages/muelu/research/mmayr/composite_to_regions/src/SetupRegionHierarchy_def.hpp

```diff
@@ -1048,87 +1047,95 @@ void vCycle(const int l, ///< ID of current level
 
     RCP<TimeMonitor> tm = rcp(new TimeMonitor(*TimeMonitor::getNewTimer("vCycle: * - coarsest grid solve")));
 
-    // First get the Xpetra vectors from region to composite format
-    // (the coarseCompMat should already exist)
-    RCP<Vector> compX = VectorFactory::Build(coarseCompMat->getRowMap(), true);
-    RCP<Vector> compRhs = VectorFactory::Build(coarseCompMat->getRowMap(), true);
-    {
-      for (int j = 0; j < maxRegPerProc; j++) {
-        RCP<Vector> inverseInterfaceScaling = VectorFactory::Build(regInterfaceScalings[l][j]->getMap());
-        inverseInterfaceScaling->reciprocal(*regInterfaceScalings[l][j]);
-        fineRegB[j]->elementWiseMultiply(SC_ONE, *fineRegB[j], *inverseInterfaceScaling, SC_ZERO);
-      }
-
-      regionalToComposite(fineRegB, compRhs, maxRegPerProc, quasiRegRowMaps[l],
-                          regRowImporters[l], Xpetra::ADD);
+    const bool useCoarseSmoother = false;
+    if (useCoarseSmoother) {
```

Comment:
It looks like this code branch is never used since `useCoarseSmoother` is set to `false`, would it make more sense to have the value come from the command line with a default value `false` instead?

Reply:
@lucbv I think you’re right. Still, this is the coarse level and we usually just Wanne use a direct solver, no smoother. This is still a weird logic here, but it doesn’t matter in practice. Let’s talk about it in our next meeting.